cruise.umple/test/cruise/umple/implementation/php/BasicPostcondition1.php.txt

```diff
@@ -1,6 +1,6 @@
 <?php
 /*PLEASE DO NOT EDIT THIS CODE*/
-/*This code was generated using the UMPLE @UMPLE_VERSION@ modeling language!*/
+/*This code was generated using the UMPLE 1.25.0-963d2bd modeling language!*/
```

Comment:
You don't need to specify the version. Please follow other test cases. We never put the version in our test cases.